docs/adr/0000-use-markdown-architectural-decision-records.md

```diff
@@ -1,23 +1,25 @@
 # Use Markdown Architectural Decision Records
 
-Should we record the architectural decisions made in this project?
-And if we do, wow to structure these recordings?
+## Context and Problem Statement
 
-## Considered Alternatives
+We want to record architectural decisions made in this project.
+Which format and structure should these records follow?
 
-* [MADR](https://adr.github.io/madr/) - Markdown Architectural Decision Records
-* [Michael Nygard's template](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions) - The first incarnation of the term "ADR". Maintainable by [adr-tools](https://github.com/npryce/adr-tools).
+## Considered Options
+
+* [MADR](https://adr.github.io/madr/) 2.0.3 - The Markdown Architectural Decision Records
+* [Michael Nygard's template](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions) - The first incarnation of the term "ADR"
 * [Sustainable Architectural Decisions](https://www.infoq.com/articles/sustainable-architectural-design-decisions) - The Y-Statements
-* [DecisionRecord](https://github.com/schubmat/DecisionCapture) - Agile records by [@schubmat](https://github.com/schubmat/)
 * Other templates listed at <https://github.com/joelparkerhenderson/architecture_decision_record>
-* No records
+* Formless - No conventions for file format and structure
 
 ## Decision Outcome
 
-* Chosen Alternative: MADR
+Chosen option: "MADR 2.0.3", because
```

Comment:
Isn't this somehow a recursive definition that the decision to use MADR is documented using MADR,.. 😏 ?

Reply:
Therefore, it's the ADR-0000 also contained in the template itself. It your own dog food 👯